bluepysnap/schemas/node/biophysical.yaml

```diff
@@ -0,0 +1,25 @@
+title: Nodes - biophysical
+description: schema for biophysical node types
+properties:
+  nodes:
+    allOf: [{ $ref: "#/$node_file_defs/nodes" }]
+    patternProperties:
+      "":
```

Comment:
why is the empty string used here?

Reply:
It's used as a wild card for the population name. Currently, any population name would suffice. We could create some rules for population names, such as 
* accept only `a-Z`, `0-9` and `_`

but those would need to be agreed upon.

Reply:
icic, can you add a comment (including in the other cases where an empty pattern is used)?